### PR TITLE
Documentation: add documentation about Intel DGPU media feature L1 and L2 tests

### PR DESCRIPTION
**Pull request by mregmi**
…d L2 tests

### CODE REVIEW

**Review by vbedida79 — APPROVED** [review]

File: tests/l2/dgpu/README.md [hunk 1]
```diff
@@ -87,5 +87,168 @@ $ oc logs intel-dgpu-hwinfo-44k4d
   Attached to: #89 (PCI bridge)
 ```                        
 
+### Verify Media support for Intel® Data Center GPU
+It is only supported for Intel® Data Center GPU Flex Series.
+
```
Comment (uMartinXu) [inline tests/l2/dgpu/README.md:92]: It is verified on Intel® Data Center GPU Flex Series.

File: tests/l2/dgpu/README.md [hunk 2]
```diff
@@ -87,5 +87,168 @@ $ oc logs intel-dgpu-hwinfo-44k4d
...
+NOTE: Please note that the Intel media driver (https://github.com/intel/media-driver) library is compiled while building the image and contains implementations of various Media codecs. Those codecs are patented and are owned by their respective developers.
+
```
Comment (uMartinXu) [inline tests/l2/dgpu/README.md:94]: this note is not so clear to me. Could you show me the original doc for the related information? Thanks!

File: tests/l2/dgpu/README.md [hunk 3]
```diff
@@ -87,5 +87,168 @@ $ oc logs intel-dgpu-hwinfo-44k4d
   Attached to: #89 (PCI bridge)
 ```                        
 
+### Verify Media support for Intel® Data Center GPU
```
Comment (uMartinXu) [inline tests/l2/dgpu/README.md:90]: Verify the Media feature provisioning for Intel® Data Center GPU

File: tests/l2/dgpu/README.md [hunk 4]
```diff
@@ -87,5 +87,168 @@ $ oc logs intel-dgpu-hwinfo-44k4d
...
+It is only supported for Intel® Data Center GPU Flex Series.
+
+NOTE: Please note that the Intel media driver (https://github.com/intel/media-driver) library is compiled while building the image and contains implementations of various Media codecs. Those codecs are patented and are owned by their respective developers.
+
+#### vainfo
```
Comment (uMartinXu) [inline tests/l2/dgpu/README.md:95]: Using VAInfo

File: tests/l2/dgpu/README.md [hunk 5]
```diff
@@ -87,5 +87,168 @@ $ oc logs intel-dgpu-hwinfo-44k4d
...
+This workload runs [vainfo](https://github.com/intel/libva-utils) utilizing the i915 resource from GPU provisioning and displays the Media fetures supported by the GPU.
```
Comment (uMartinXu) [inline tests/l2/dgpu/README.md:96]: features

File: tests/l2/dgpu/README.md [hunk 6]
```diff
@@ -87,5 +87,168 @@ $ oc logs intel-dgpu-hwinfo-44k4d
...
+* Check the results.
+```
+  $ oc get pods -n intel-dgpu
+  NAME                        READY   STATUS      RESTARTS   AGE
+  intel-dgpu-vainfo-1-build   0/1     Completed   0          9m49s
+  intel-dgpu-vainfo-rbdvz     0/1     Completed   0          45s
+```
+```
+$ oc logs intel-dgpu-vainfo-rbdvz -n intel-dgpu
+libva info: VA-API version 1.20.0
```
Comment (uMartinXu) [inline tests/l2/dgpu/README.md:119]: please remove line 119 to 123

File: tests/l2/dgpu/README.md [hunk 7]
```diff
@@ -87,5 +87,168 @@ $ oc logs intel-dgpu-hwinfo-44k4d
...
+      VAProfileHEVCSccMain10          : VAEntrypointVLD
+      VAProfileHEVCSccMain10          : VAEntrypointEncSliceLP
+      VAProfileHEVCSccMain444         : VAEntrypointVLD
+      VAProfileHEVCSccMain444         : VAEntrypointEncSliceLP
+      VAProfileAV1Profile0            : VAEntrypointVLD
+      VAProfileAV1Profile0            : VAEntrypointEncSliceLP
+      VAProfileHEVCSccMain444_10      : VAEntrypointVLD
+      VAProfileHEVCSccMain444_10      : VAEntrypointEncSliceLP
+```
+#### libvpl
```
Comment (uMartinXu) [inline tests/l2/dgpu/README.md:167]: Using libvpl

File: tests/l2/dgpu/README.md [hunk 8]
```diff
@@ -87,5 +87,168 @@ $ oc logs intel-dgpu-hwinfo-44k4d
...
+This workload runs various test programs from [libvpl](https://github.com/intel/libvpl) utilizing the i915 resource from GPU provisioning and displays the VPL fetures supported by the GPU.
```
Comment (uMartinXu) [inline tests/l2/dgpu/README.md:168]: features 
Video Processing Library (VPL)

File: tests/l2/dgpu/README.md (hunk 6 above)
Comment (mregmi) [reply tests/l2/dgpu/README.md:119]: why not show the exact logs it produces?

File: tests/l2/dgpu/README.md (hunk 8 above)
Comment (mregmi) [reply tests/l2/dgpu/README.md:168]: Done

File: tests/l2/dgpu/README.md (hunk 7 above)
Comment (mregmi) [reply tests/l2/dgpu/README.md:167]: Done

File: tests/l2/dgpu/README.md (hunk 5 above)
Comment (mregmi) [reply tests/l2/dgpu/README.md:96]: Done

File: tests/l2/dgpu/README.md (hunk 4 above)
Comment (mregmi) [reply tests/l2/dgpu/README.md:95]: Done

File: tests/l2/dgpu/README.md (hunk 3 above)
Comment (mregmi) [reply tests/l2/dgpu/README.md:90]: Done

File: tests/l2/dgpu/README.md (hunk 2 above)
Comment (mregmi) [reply tests/l2/dgpu/README.md:94]: Not sure i understand the list of supported codecs are in the main readme of the project above. but if its confusing we can remove this note

File: tests/l2/dgpu/README.md (hunk 1 above)
Comment (mregmi) [reply tests/l2/dgpu/README.md:92]: Done

File: tests/l2/dgpu/README.md (hunk 2 above)
Comment (uMartinXu) [reply tests/l2/dgpu/README.md:94]: We should add the notes, Just need to make it very concise and clear. :-)

Comment (mregmi) [reply tests/l2/dgpu/README.md]: removed that line as its confusing.

File: tests/l2/dgpu/README.md (hunk 6 above)
Comment (mregmi) [reply tests/l2/dgpu/README.md]: removed those lines.

**Review by uMartinXu — APPROVED** [review]

**Pull request merged**